app/models/piece.rb

```diff
@@ -67,4 +68,9 @@ def path_obstructed?(x, y)
     return diagonal_obstructed?(x, y) if diagonal_move?(x, y)
     false
   end
+
+  def opposing_color
```

Comment:
(Tyna was in the middle of changing how the piece color is represented and I didn't know how it would end up being represented.)